pkg/gatherers/clusterconfig/gather_node_logs.go

```diff
@@ -105,24 +118,44 @@ func nodeLogString(ctx context.Context, req *rest.Request) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	defer in.Close()
+	defer func() {
+		_ = in.Close()
+	}()
 
 	r, err := gzip.NewReader(in)
 	var scanner *bufio.Scanner
 	if err != nil {
 		scanner = bufio.NewScanner(in)
 	} else {
-		defer r.Close()
+		defer func() {
+			_ = r.Close()
+		}()
 		scanner = bufio.NewScanner(r)
 	}
 
-	messagesToSearch := []string{
-		"E\\d{4} [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}", //  Errors from log
-	}
+	messagesToSearch := nodeLogsMessagesFilter()
 	return common.FilterLogFromScanner(scanner, messagesToSearch, true, func(lines []string) []string {
 		if len(lines) > logNodeMaxLines {
 			return lines[len(lines)-logNodeMaxLines:]
 		}
 		return lines
 	})
 }
+
+func nodeLogsMessagesFilter() []string {
+	return []string{
+		"E\\d{4} [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}", //  Errors from log
+		`connect: connection refused`,
+		`failed (failure): command timed out`,
+		`Failed to make webhook authenticator request: Post`,
+		`raise JSONDecodeError("Expecting value", s, err.value) from None`,
+		`ContainerStateWaiting{Reason:ContainerCreating`,
+		`ContainersNotReady Message:containers with unready status`,
+		`MountVolume.MountDevice failed for volume`,
+		`kubernetes.io/csi: attacher.MountDevice failed to create newCsiDriverClient`,
+		`Unable to attach or mount volumes: unmounted volumes`,
+		`timed out waiting for the condition`,
+		`CreateContainerError: context deadline exceeded`,
+		`rpc error: code = ResourceExhausted desc = grpc: received message larger than max`,
+	}
+}
```

Comment:
Minor: perhaps it would be slightly better to move this to some constant or share variable instead of creating the slice multiple times in the for loop (for each node)

Reply:
That is a good point. I considered doing that but didn't want to leave it as a constant on the package scope. It should be called just once at the right before the loop to create a variable, then the GC can handle it, but it would require some refactoring - and adjusting unit tests as well. Not sure if I want to address it on this PR.